infrastructure/application/index.ts

```diff
@@ -279,6 +280,9 @@ export = async () => {
     sslPolicy: "ELBSecurityPolicy-TLS-1-2-Ext-2018-06",
     certificateArn: certificates.requireOutput("certificateArn"),
   });
+  const fileApiKey = new random.RandomPassword("file-api-key", {
```

Comment:
Can these specific rotation docs also perhaps be added to the Pulumi/infrastructure/application README? Definitely a useful one I don't want to lose track of in this PR once merged.

Reply:
additional question here in the context of BOPS (I was trying to share the secret with the devs during the call today): doesn't seem like `pulumi config get file-api-key --stack staging` here works as normal - is there an alternative way to get this?

Reply:
I'm exporting this variable so that would be `pulumi stack output --show-secrets fileApiKeys`.

However, this isn't staging. This is a pizza, so the secret is hard-coded.